engine/src/main/java/org/terasology/world/chunks/pipeline/ChunkTask.java

```diff
@@ -7,10 +7,6 @@
 import org.terasology.utilities.concurrency.Task;
 import org.terasology.world.chunks.Chunk;
 
-/**
- * @deprecated Use {@link org.terasology.world.chunks.pipeline.stages.ChunkTask} instead
- */
-@Deprecated
```

Comment:
So we reverted the deprecation? Was the other `ChunkTask` recently introduced, or has it been around for some time but was never put to "special" use (justifying this deprecation)?